feat: update docs and compress images (#5531)
rotki/rotki: docs/usage_guide.rst

Liquity Staking
===============

> If you stake LQTY in the protocol you can see stability pool deposits and staked amount, and the stake events.

Reviewer: ```suggestion
If you stake LQTY in the protocol you can see stability pool deposits, staked amount, and the stake events.
```

2. The type of custom asset being represented. It's just a string. The type field remembers all previously used types. This is required.
3. The note to be added to the custom asset. This is optional.

Newly detected assets
=====================

> You can inspect the list of newly detected tokens and add any spam assets to the ignored tokens list.

Reviewer: Explain better that this needs to be inspected often.

```suggestion
All newly detected EVM tokens will appear in the list of newly detected assets. You should inspect this list often and accept valid tokens and reject spam assets by adding them to the ignored tokens list.
```

.. image:: images/rotki_manage_newly_detected_assets.png
   :alt: Manage newly detected assets
   :align: center

Merging two assets
=======================

There are two possible situations where you might need to merge two assets into one.

> 1. You added a custom asset that was later officially supported on rotki. In this case you should merge your custom asset to the officially supported one. If you don't do this might split balances between entries, especially for supported exchanges that will use the officially supported entry.

Reviewer: ```suggestion
1. You added a custom asset that was later officially supported on rotki. In this case you should merge your custom asset with the officially supported one. If you don't do this, you will see split balances between entries, especially for supported exchanges that will use the officially supported entry.
```
